libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFAction.h

```diff
@@ -202,6 +202,20 @@
 
 - (BOOL)shouldRetryWithError:(NSError*)error;
 
+/** Subclassable method that will be called when the action is about to call responseBlock. 
+ 
+ @discussion
```

Comment:
We might want to make note that you must call the super class version of this as well, since it's responsible for cleaning up the operations.  Otherwise, :+1: